eva_cttv_pipeline/clinvar_to_evidence_strings.py

```diff
@@ -24,6 +24,33 @@
 TMPDIR = '/tmp/'
 
 
+def write_output(dir_out, nsv_list, unmapped_traits, unavailable_efo_dict, evidence_string_list, evidence_list):
```

Comment:
I think it is not necessary to declare this function before `clinvar_to_evidence_strings`: it won't be interpreted until that function is called and by then it will have been already parsed. Placing it after the main function will make the code flow more obvious.